### PR TITLE
💄 Use loading indicator in civic redirect page

### PR DESCRIPTION
**Pull request by williamchong007**
image

### CODE REVIEW

File: pages/in/civic/index.vue [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
-  <div>{{$t('General.loading')}}</div>
+  <div class="lc-container-0">
+    <div class="lc-container-1">
+      <div class="lc-container-2">
+        <div class="lc-container-3">
+          <lc-loading-indicator />
+        </div>
+      </div>
+    </div>
+  </div>
```
Comment (nnkken) [inline pages/in/civic/index.vue]: @nwingt please comment this, I don't know why the containers are necessary

**Comment by nwingt** [comment]
I have opened a PR williamchong007/likecoin-store#2

File: pages/in/civic/index.vue (hunk 1 above)
Comment (nwingt) [reply pages/in/civic/index.vue]: Updated

**Comment by williamchong007** [comment]
could have just PR ur version directly to this repo

**Comment by nwingt** [comment]
Duplicated #1103

**Pull request closed without merging**